index.html

```diff
@@ -1,173 +1,225 @@
 
-		                </div>
+		<section class="image-slider slider-all-controls controls-inside pt0 pb0 height-70 parallax">
+			<ul class="slides">
+				<li class="overlay image-bg">
+					<div class="background-image-holder">
+						<img alt="image" class="background-image" src="img/slider/community-map.jpg">
+					</div>
+					<div class="container v-align-transform">
+						<div class="row text-center">
+							<div class="col-md-10 col-md-offset-1">
+								<h2 class="mb-xs-16">Educating and inspiring students by being
+									<br class="hidden-sm"> involved in open source projects</h2>
+								<p class="lead mb40">
+									Participate in Google Codein with
+									<b>FOSSASIA</b> !
+								</p>
+								<a class="btn btn-lg btn-filled" href="https://codein.withgoogle.com/organizations/fossasia/">Take me Along !</a>
+								<a class="btn btn-lg" href="#mentors">Explore the site</a>
+							</div>
+						</div>
 
-		            </li><li class="overlay image-bg">
-		                <div class="background-image-holder">
-		                    <img alt="image" class="background-image" src="img/slider/group-photo.jpg">
-		                </div>
-		                <div class="container v-align-transform ">
-		                    <div class="row text-center">
-		                        <div class="col-md-10 col-md-offset-1">
-		                            <h2 class="mb-xs-16"><b><i>"The Open Conversational Web"</i></b>  <br class="hidden-sm">Asia's Premier Open Technology Event</h2>
-		                            <p class="lead mb40">
-		                                FOSSASIA SUMMIT IN SINGAPORE<br>
-		                                22<sup>th</sup> - 25<sup>th</sup> March, 2018
-		                            </p>
-		                            <a class="btn btn-lg btn-filled" href="https://2018.fossasia.org/">I'm Interested !</a>
-		                        </div>
-		                    </div>
+					</div>
 
-		                </div>
+				</li>
+				<li class="overlay image-bg">
+					<div class="background-image-holder">
+						<img alt="image" class="background-image" src="img/slider/group-photo.jpg">
+					</div>
+					<div class="container v-align-transform ">
+						<div class="row text-center">
+							<div class="col-md-10 col-md-offset-1">
+								<h2 class="mb-xs-16">
+									<b>
+										<i>"The Open Conversational Web"</i>
+									</b>
+									<br class="hidden-sm">Asia's Premier Open Technology Event</h2>
+								<p class="lead mb40">
+									FOSSASIA SUMMIT IN SINGAPORE
+									<br> 22
+									<sup>th</sup> - 25
+									<sup>th</sup> March, 2018
+								</p>
+								<a class="btn btn-lg btn-filled" href="https://2018.fossasia.org/">I'm Interested !</a>
+							</div>
+						</div>
+
+					</div>
+
+				</li>
+			</ul>
+		</section>
 
-		            </li>
-		        </ul>
-		    </section><section class="bg-secondary" id="mentors">
+		<section class="bg-secondary" id="mentors">
+<<<<<<< HEAD
```

Comment:
Please fix the merge conflict. There are many places this needs to be fixed, please take a look.

Thanks!